cmd/bridge/main.go

```diff
@@ -107,6 +107,14 @@ func main() {
 	fStatuspageID := fs.String("statuspage-id", "", "Unique ID assigned by statuspage.io page that provides status info.")
 	fDocumentationBaseURL := fs.String("documentation-base-url", "", "The base URL for documentation links.")
 
+	fMonitoringAlermanagerURL := fs.String("monitoring-alermanager-url", "", "Alertmanager monitoring URL.")
+	fMonitoringGrafanaURL := fs.String("monitoring-grafana-url", "", "Grafana monitoring URL.")
+	fMonitoringPrometheusURL := fs.String("monitoring-prometheus-url", "", "Prometheus monitoring URL.")
+	fMonitoringThanosURL := fs.String("monitoring-thanos-url", "", "Thanos monitoring URL.")
+
+	fLoggingKibanaAppURL := fs.String("logging-kibana-app-url", "", "KibanaApp logging URL.")
```

Comment:
`kibana-app-url` URL, etc

```suggestion
	fKibanaURL := fs.String("kibana-url", "", "Public Kibana URL for non-infrastructure namespaces.")
```